src/D2L.Bmx/BmxConfigProvider.cs

```diff
@@ -53,6 +54,28 @@ internal class BmxConfigProvider : IBmxConfigProvider {
 		);
 	}
 
+	public void SaveConfiguration( BmxConfig config ) {
+		using StreamWriter writer = new StreamWriter( ConfigFileName(), false );
+
+		if( !string.IsNullOrEmpty( config.Org ) ) {
+			writer.WriteLine( $"org={config.Org}" );
+		}
+		if( !string.IsNullOrEmpty( config.User ) ) {
+			writer.WriteLine( $"user={config.User}" );
+		}
+		if( config.DefaultDuration.HasValue ) {
+			writer.WriteLine( $"default_duration={config.DefaultDuration}" );
+		}
+	}
+
+	internal static string ConfigFileName() {
+		return Path.Join( UserHomeDir(), ".bmx", "config" );
+	}
+
+	internal static string UserHomeDir() {
```

Comment:
```suggestion
	private static string UserHomeDir() {
```

Reply:
I don't think this is the only place where we have this value? we might consider consolidating them.
can do this later.